### DISCUSSION TITLE
Cync Lights?

### DISCUSSION THREAD

**Discussion by miketunes** [basnijholt/adaptive-lighting#1134] [Q&A]
I have govee and Wiz lights working successfully but I'm using this integration and they don't change brightness or temp:
https://github.com/nikshriv/cync_lights

**Comment by poolguard** [comment]
I have a number of the Cync lights(under counter, bulbs, and puck lights).  using matter they all work as expected.  i was never able to get a bunch of things to work correctly with that addin.
